packages/contentstack-branch/package.json

```diff
@@ -6,7 +6,8 @@
   "bugs": "https://github.com/contentstack/cli/issues",
   "dependencies": {
     "@contentstack/cli-command": "^1.2.0",
-    "@contentstack/cli-utilities": "^1.4.0",
+    "@contentstack/cli-utilities": "^1.2.0",
```

Comment:
pls use     "@contentstack/cli-utilities": "^1.4.0",
merge what is incoming herer

Reply:
"@contentstack/management": "^1.6.1",
remove this from here